ext/oci8/oci8.stub.php

```diff
@@ -252,13 +252,22 @@ function ocifetchstatement($statement, &$output, int $offset = 0, int $limit = -
 /** @param resource $statement */
 function oci_fetch_object($statement, int $mode = OCI_ASSOC | OCI_RETURN_NULLS): stdClass|false {}
 
-/** @param resource $statement */
+/**
+ * @param resource $statement
+ * @return array<int|string, mixed>|false
+ */
 function oci_fetch_row($statement): array|false {}
 
-/** @param resource $statement */
+/**
+ * @param resource $statement
+ * @return array<int|string, mixed>|false
+ */
 function oci_fetch_assoc($statement): array|false {}
 
-/** @param resource $statement */
+/**
+ * @param resource $statement
+ * @return array<int|string, mixed>|false
```

Comment:
These don't look particularly useful...

Reply:
I'm not sure why I added them in the first place - most probably in order to exclude `ref` from the possible array values.